Dockerfile

```diff
@@ -3,28 +3,33 @@ FROM golang:1.17-bullseye AS builder
 
 WORKDIR /workspace
 
-# Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
+RUN ["go",  "mod", "download"]
```

Comment:
I think that's a welcome change. Could we have a comment explaining the order of instructions? Now it might seem weird to a developer opening the file for the first time.

Reply:
You mean comment inside Dockerfile?

Reply:
Yes, I would add one there.

Reply:
Please take a look at comment that is on line 6 - is that sufficient?

Reply:
Yes, it is great. Thank you!